catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -242,8 +243,14 @@ public void validate(Metacard metacard) throws ValidationException {
             schematronReport = generateReport(metadata, validator);
             if (!schematronReport.isValid(suppressWarnings)) {
                 throw new SchematronValidationException("Schematron validation failed.",
-                        schematronReport.getErrors(),
-                        schematronReport.getWarnings());
+                        schematronReport.getErrors()
+                                .stream()
+                                .map(error -> sanitize(error))
```

Comment:
Looks like you could use a method reference here.

http://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html